ui/admin/app/components/form/host-set/azure/index.hbs

```diff
@@ -74,11 +74,10 @@
     </:body>
   </form.fieldset>
 
-  <Rose::Form::Fieldset>
-    <:title>{{t 'resources.host-set.form.preferred_endpoints.label'}}</:title>
-    <:description>{{t
+  <Rose::Form::Fieldset @helperText={{t
         'resources.host-set.form.preferred_endpoints.help'
-      }}</:description>
+      }} @linkText={{t 'actions.learn-more'}} @link={{doc-url 'host-set.preferred-endpoints'}}>
+    <:title>{{t 'resources.host-set.form.preferred_endpoints.label'}}</:title>
```

Comment:
I think these files need to be reformatted

Reply:
hmm, I did yarn lint:hbs --fix like I usually do (which didn't pick up this). Do you recommend anything different? 

`yarn format` gives me 210 files 😓

Reply:
Ugh yeah unfortunately I don't think `template-lint` fixes these type of format issues. I still have prettier running automatically in my IDE which is probably why it fixes it. Unfortunately, the `yarn format` will still strip newlines away, (but theoretically if you commit it `template-lint` will add the new lines again...which is not a good workflow).

I don't have a good answer for you besides running prettier 😿, perhaps we should discuss this in tech time again since `yarn format` is somewhat unusable now? I didn't notice since I had never used that as part of my workflow

Reply:
apparently, my text editor didn't automatically turn on prettier formatting on save. I did it for the files in this PR but I'm afraid there might be other files in the repo that weren't formatted properly. Open to discussing this in tech time!